xml/System.Net/HttpWebRequest.xml

```diff
@@ -1445,7 +1445,7 @@ Both <xref:System.Net.HttpWebRequest> constructors are obsolete and should not b
 >  Your application cannot mix synchronous and asynchronous methods for a particular request. If you call the <xref:System.Net.HttpWebRequest.BeginGetRequestStream%2A> method, you must use the <xref:System.Net.HttpWebRequest.BeginGetResponse%2A> method to retrieve the response.  
   
 > [!NOTE]
->  This member outputs trace information when you enable network tracing in your application. For more information, see [&#91;\<topic://conUsingNetworkTracing>&#93;](https://msdn.microsoft.com/library/e993b7c3-087f-45d8-9c02-9dded936d804).  
+>  This member outputs trace information when you enable network tracing in your application. For more information, see [Network Tracing in the .NET Framework](https://docs.microsoft.com/dotnet/framework/network-programming/network-tracing).  
```

Comment:
Let's use the ~ links to get build time validation and have the links work on offline books as well

```suggestion
>  This member outputs trace information when you enable network tracing in your application. For more information, see [Network Tracing in the .NET Framework](~/docs/framework/network-programming/network-tracing.md).  
```